library/fsharp

```diff
@@ -1,5 +1,14 @@
-# maintainer: Henrik Feldt <henrik@haf.se> (@haf)
+Maintainers: Henrik Feldt <henrik@haf.se> (@haf)
+GitRepo: https://github.com/fsprojects/docker-fsharp.git
 
-4.0.0.4: git://github.com/fsprojects/docker-fsharp@57b3ddadc85a63bf586ab89b523bb8326dad9c12 4.0.0.4
-4.0.1.1: git://github.com/fsprojects/docker-fsharp@57b3ddadc85a63bf586ab89b523bb8326dad9c12 4.0.1.1
-latest: git://github.com/fsprojects/docker-fsharp@57b3ddadc85a63bf586ab89b523bb8326dad9c12 4.0.1.1
+Tags: latest, 4, 4.1, 4.1.0.1
+GitCommit: 289a5e0066c69255c055dd5ee22f2e583d37fab4
+Directory: 4.1.0.1
+
+Tags: 4.0, 4.0.1, 4.0.1.1
+GitCommit: 289a5e0066c69255c055dd5ee22f2e583d37fab4
+Directory: 4.0.1.1
+
+Tags: 4.0.0.4
```

Comment:
Did you want to leave this version in?  Is it still supported?  Removing it here will remove it from the "Supported" section on the Hub readme (and will prevent us from spending cycles rebuilding it on the official build server), but the tag will still be available to users who want it.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)